dir/path.go

```diff
@@ -69,27 +89,38 @@ func (p *PathManager) Localkey(name string) (keyPath, certPath string) {
 	return keyPath, certPath
 }
 
-// SigningKeyConfig return the path of signingkeys.json files
+// SigningKeyConfig returns the writable user level path of signingkeys.json files
 func (p *PathManager) SigningKeyConfig() string {
 	path, err := p.UserConfigFS.GetPath(SigningKeysFile)
 	checkError(err)
 	return path
 }
 
-// TrustPolicy returns the path of trustpolicy.json file
+// TrustPolicy returns the ready-only path of trustpolicy.json file
 func (p *PathManager) TrustPolicy() string {
 	path, err := p.ConfigFS.GetPath(TrustPolicyFile)
```

Comment:
The path of trystpolicy file is know(based on system or user level) then why are we searching that in directory using `GetPath`?

Reply:
GetPath will search both `user` and `system` level, and user level has higher priority, so the GetPath helps search and decide the priority.